snippets/javascript/statements.md

```diff
@@ -0,0 +1,50 @@
+# JS Statements
+
+## If statement
+
+```js
+const isMailSent = true;
+
+if (isMailSent) {
+  console.log("Mail sent to recipient");
+}
+```
+
+<!-->An if statement accepts an expression with a set of parentheses:
```

Comment:
I think the `<!-->` in your MD file are a mistake, or are you trying to achieve something with them? 😄  There are several of them, so please remove/change them all.